src/main/java/seedu/address/model/VersionedResumeBook.java

```diff
@@ -94,31 +94,35 @@ public boolean equals(Object other) {
 
         // state check
         return super.equals(otherVersionedResumeBook)
-                && resumeBookStateList.equals(otherVersionedResumeBook.resumeBookStateList)
+                //&& resumeBookStateList.equals(otherVersionedResumeBook.resumeBookStateList)
```

Comment:
Did you mean to uncomment this?

Reply:
Oh no :(
My bad.
I will fix it asap and check for any potential arising issue.